packages/astro/src/assets/fonts/providers/utils.ts

```diff
@@ -13,18 +14,25 @@ export function resolveEntrypoint(root: URL, entrypoint: string): string {
 	}
 }
 
-export function validateMod(mod: any): Pick<ResolvedFontProvider, 'provider'> {
+export function validateMod(
+	mod: any,
+	providerName: string,
+): Pick<ResolvedFontProvider, 'provider'> {
 	try {
 		if (!mod.provider && typeof mod.provider !== 'function') {
-			// TODO: improve
-			throw new Error('Not a function');
+			throw new Error('provider is not a function');
```

Comment:
Let's provide more information about how a user should fix this error.

Reply:
This is used as `cause` of an `AstroError` so I'm not sure if telling the user how to fix it should live here. But maybe the error message could be improved, eg:
```suggestion
			throw new Error('The "provider" export is not a function');
```

Reply:
Then leave a comment that explains why we don't use an astro error here